hs_core/views/resource_rest_api.py

```diff
@@ -620,6 +622,31 @@ def put(self, request, pk):
             shutil.rmtree(tmp_dir)
 
 
+class ResourceMapRetrieve(APIView):
+    """
+    Retrieve resource map
+
+    REST URL: hsapi/resmap/{pk}
+    HTTP method: GET
+
+    :type pk: str
+    :param pk: id of the resource
+    :return: science metadata as XML document
```

Comment:
@alvacouch return resourcemap as XML document, right? Looks like a copy and paste typo in the comment.